src/devtools/client/inspector/layout/components/LayoutApp.tsx

```diff
@@ -5,13 +5,12 @@
 import React from "react";
 import { ConnectedProps, connect } from "react-redux";
 
+import Accordion from "devtools/client/shared/components/Accordion";
+import { prefs as servicePrefs } from "devtools/shared/services";
 import type { UIState } from "ui/state";
 
 import { BoxModel } from "../../boxmodel/components/BoxModel";
 
-const Services = require("devtools/shared/services");
-
-const Accordion = require("devtools/client/shared/components/Accordion");
```

Comment:
This was the bug. `Accordion` is a _default export_ but we don't have `.default` on the end, so we're trying to render _the module itself_.

I assume it's okay to just use `import` statements here instead of `require`? Dunno why we were doing the other to begin with.

Reply:
IIRC this was necessary to import javascript (instead of typescript) modules, perhaps because we didn't have `allowJs` set in our tsconfig. It doesn't seem to be necessary anymore.